samples/benchmarks/coremark/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Zephyr Kernel"
+	source "Kconfig.zephyr"
+endmenu
+
+menu "CoreMark sample"
+
+config APP_MODE_FLASH_AND_RUN
+	bool "Run CoreMark benchmark on start up"
+	help
+	  If enabled, CoreMark will start execution immediately after CPU start up.
+	  Otherwise, it will wait button to be pressed.
+
+endmenu
+
+config BOARD_ENABLE_CPUNET_HELPER
+	bool "Include net image as sub image"
+	depends on SOC_NRF5340_CPUAPP
+	default y if (BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS)
+	select BOARD_ENABLE_CPUNET
```

Comment:
Can we enable the `BOARD_ENABLE_CPUNET` option in a dedicated board configuration instead of having this option?

Reply:
We could. I think Vova created it here so there was no extra conf file with only this option. I can change if you prefer that way

Reply:
yes, please do